resources/generate_book.py

```diff
@@ -462,6 +490,48 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
     ))
 
     book.category('mechanics', 'Advanced Mechanics', 'Advanced sections of the tech tree, from the first pickaxe, all the way to colored steel.', 'tfc:metal/axe/red_steel', entries=(
+        entry('animal_husbandry', 'Animal Husbandry', 'minecraft:egg', pages=(
+            text('$(thing)Livestock() are animals that can be tamed and bred by the player. Livestock can be either $(thing)male$() or $(thing)female$(). For some animals, it is possible to tell their sex visually. For example, male pigs have tusks.'),
+            text('Livestock experience $(thing)aging$(). They are born as babies, which are smaller and cannot provide things for the player. After a certain amount of days, they grow into $(thing)adult$() animals, which are able to do things like breed or produce milk. After they breed or are used enough times, animals become $(thing)old$(), and are only useful for their meat.'),
+            image('tfc:textures/gui/book/tutorial/old_cow.png', text_contents='This bull is old and cannot breed, because of its faded coat and grey, unseeing eyes.'),
+            text('Livestock can be fed to raise $(thing)familiarity$(). Each animal has a set of food that it prefers to eat. To feed an animal, $(item)$(k:key.sneak)$() and $(item)$(k:key.use)$() with food in your hand.'),
+            image('tfc:textures/gui/book/tutorial/unfamiliarized_pig.png', text_contents='Livestock have a $(thing)familiarity indicator$() that shows how familiar they are with you. It is visible when holding $(item)$(k:key.sneak)$() and looking at the animal.'),
+            image('tfc:textures/gui/book/tutorial/no_familiarity_decay_pig.png', text_contents='Familiarity decays each day by a little, unless it reaches a certain amount, indicated by the heart being outlined in white.'),
+            image('tfc:textures/gui/book/tutorial/familiarity_limit_pig.png', text_contents='Adult livestock cannot be familiarized above a certain amount, indicated by the red outlined heart. Babies can reach up to 100% familiarity.'),
+            text('$(thing)Mammals$() are livestock that experience $(thing)pregnancy$(). Adult mammals that are above 30% familiarity and have been fed that day will mate, given that they are of opposite genders and near each other. The female animal will become $(thing)pregnant$(), which causes it to spawn children a set number of days after fertilization.'),
+            text('$(thing)Wooly Animals$() are mammals that can be $(thing)sheared$() if they are adults and familiar enough to you. Shearing an animal makes it age, but not as much as breeding does.'),
+            text('$(thing)Dairy Animals$() are mammals that make $(thing)milk$(). Female dairy animals can be clicked with a bucket to obtain milk. Milking makes an animal age, but not as much as breeding does.'),
```

Comment:
As much as the internal mechanic works (i.e. "uses", not "age"), I don't like directly referring to "shearing an animal makes it age", because that's just a bit weird. I think the less we talk about the specifics of aging, and more alluding to how "Animals get older over time, and being interacted with", or "An animal can only be milked/bred/ so many times during it's adult life" would be better immersion wise.